index.html

```diff
@@ -28,19 +28,22 @@
   </header>
 
   <section class="search-section">
-    <input type="text" placeholder="Search software jobs" />
-    <button class="search-button">
-      <span class="fa fa-search"></span>
-    </button>
+    <form action="#">
+      <label for="search">Search Jobs</label>
+      <input type="text" id="search" placeholder="Search for a software job" />
```

Comment:
Eh, I disagree. It shouldn't be the same as the label and it should be phrased to prompt the user for a specific action, like searching for a single job (they can only search one at a time) rather than a general description like "Search software jobs" or "Search jobs" which should semantically be the label